ipinfo/cmd_init.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/pflag"
+	"golang.org/x/term"
+)
+
+func printHelpInit() {
+	fmt.Printf(
+		`Usage: %s init [<opts>] [<token>]
+
+Options:
+  --token <tok>, -t <tok>
+    token to login with.
+    (this is potentially unsafe; let the CLI prompt you instead).
+  --no-check
+    disable checking if the token is valid or not.
+    default: false.
+  --help, -h
+    show help.
+`, progBase)
+}
+
+type signupCli struct {
+	SignupURL string `json:"signupURL"`
+}
+
+type tokenCli struct {
+	Token string `json:"token"`
+}
+
+func cmdInit() error {
+	var num int
```

Comment:
Let's define this below where it will be used for the first time and call it `opt` instead